packages/lib-user/src/components/shared/BarChart/BarChart.js

```diff
@@ -99,16 +99,17 @@ function BarChart({
           label: typeLabel,
           render: ((number) => {
             if (type === 'session_time') {
-              const time = number / dateRangeLabel.time  
+              const time = number / dateRangeLabel.time
+              const timeLabelText = dateRangeLabel.timeLabel === 'hrs' ? `${time.toFixed(1).toLocaleString()} ${dateRangeLabel.timeLabel}` : `${time.toFixed(0).toLocaleString()} ${dateRangeLabel.timeLabel}`
```

Comment:
This can't use the convertStatsSecondsToHours utility function as sometimes it's minutes sometimes it's hours, depending on the date range. Nonetheless this change keeps the time displayed consistent with other components (with one decimal place).